### PR TITLE
[4.0] Remove sidebar from com_installer

### PR DESCRIPTION
**Pull request by brianteeman**
This pr removes the sidebar from all the views previously accessed (in J3) from the extensions menu.

As in J4 they all have their own place in the system dashboard they can simply be removed

### CODE REVIEW

**Comment by wilsonge** [comment]
Agree with this generally. Only question is do we need a back to system admin dashboard or are we happy with the browser back button?

**Comment by brianteeman** [comment]
@wilsonge i did think about that but figured it was beyond the scope of this pr as the same comment could be made about a lot of the links from the dashboard and i kind of hoped the secret template group might have already addressed that

**Comment by brianteeman** [comment]
@wilsonge Also I left the function addSubmenu in the code (but empty) I wasn't sure if it should be completely removed or not

File: administrator/components/com_installer/View/Installer/HtmlView.php [hunk 1]
```diff
@@ -112,7 +112,5 @@ protected function addToolbar()
 			ToolbarHelper::divider();
 		}
```
Comment (Quy) [inline administrator/components/com_installer/View/Installer/HtmlView.php]:
```suggestion
```

**Comment by softforge** [comment]
I have tested this item :white_check_mark: successfully on 673c79157f2bdaaf4783423f71ac4a6ac593fd5e<br /><br />Does as it says and removes the side menu in
Manage Extensions
Install Extensions
Update Extensions<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24080">issues.joomla.org/tracker/joomla-cms/24080</a>.</sub>

**Comment by Bodge-IT** [comment]
I have tested this item :white_check_mark: successfully on 673c79157f2bdaaf4783423f71ac4a6ac593fd5e<br /><br />Sidebar gone from extensions,discover & INSTALL LANG.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24080">issues.joomla.org/tracker/joomla-cms/24080</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by wilsonge** [comment]
> @wilsonge Also I left the function addSubmenu in the code (but empty) I wasn't sure if it should be completely removed or not

Kill it. It serves no purpose anymore

> @wilsonge i did think about that but figured it was beyond the scope of this pr as the same comment could be made about a lot of the links from the dashboard and i kind of hoped the secret template group might have already addressed that

They've not touched com_installer. Some of the works gone into the public repo. and most the rest has in the private repo has been on the admin menu, dashboard and login screens

**Comment by brianteeman** [comment]
removed the function  as requested

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**